bin/impl/load-env.sh

```diff
@@ -89,3 +89,11 @@ fi
 : ${ACCUMULO_USER:?"ACCUMULO_USER is not set in env.sh"}
 : ${ACCUMULO_PASSWORD:?"ACCUMULO_PASSWORD is not set in env.sh"}
 : ${ACCUMULO_TABLE:?"ACCUMULO_TABLE is not set in env.sh"}
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  export MD5=md5
```

Comment:
ok  I see.  `fluo-dev` does not source the scripts in `impl` but launches them as sup processes.